### DISCUSSION TITLE
w-[600px] = w=[] in local dev mode?

### DISCUSSION THREAD

**Discussion by yabdab** [maizzle/framework#1186] [Q&A] [answered]
When using local dev mode, direct width variants are not working. 

Everything works fine when I build production version however.

Did not always do this before.

Using Maizzle v4.7.6 and  Node v20.8.0

**Comment by cossssmin** [comment]
Just double-checked and it works fine (video using [default Starter](https://github.com/maizzle/maizzle)), can you share a repo that reproduces the issue?

https://github.com/maizzle/framework/assets/1656595/6399e289-b345-4174-a451-9d8e8be6c65a

**Reply by yabdab** [reply]
Yea it has worked fine for me in the past week actually. May just be a local issue 🤷‍♂️

I will just work around it until I can figure it out.

**Comment by gynekolog** [comment]
@yabdab 

I had a similar issue. Try to set the content in the tailwind.config.js, e.g.:
```
module.exports = {
  ...
  content: ["./src/**/*.html"],
  ...
 }
 ```

**Reply by yabdab** [reply]
Thanks, I will give that a try.

**Reply by yabdab** [reply]
Still not working. I can also use `w-2/5` and it gets parsed as `w-2/` (drops the 5) ?

**Reply by cossssmin** [reply]
If you can provide a repo that reproduces the issue I’ll be happy to take a look (make sure to include the package lock file). 

Otherwise not sure how to help since I can’t reproduce it…

**Reply by yabdab** [reply, marked as answer]
I ended up just starting over from scratch and now everything works as expected.
